apps/ledger-live-desktop/src/logger/logger-transport-main.js

```diff
@@ -1,20 +1,2 @@
-import Transport from "winston-transport";
-
-export default class MainTransport extends Transport {
-  logs = [];
-  capacity = 3000;
-  blacklist = [];
-
-  log(info, callback) {
-    setImmediate(() => {
-      this.emit("logged", info);
-    });
-
-    if (!this.blacklist.includes(info.type)) {
-      this.logs.unshift(info);
-      this.logs.splice(this.capacity);
-    }
-
-    callback();
-  }
-}
+import MemoryTransport from "./logger-transport-in-memory";
```

Comment:
couldn't we just delete this file and do the default export in `index.js` ?

Reply:
I'm not really sure to follow. I have factorized the previous logger "main" to be used both by this new "main" and by "renderer".
the benefit of this alias is abstraction: main don't need to know the technical detail if it's a "in memory".